ihontaryk/homework_8/restaurant.py

```diff
@@ -0,0 +1,78 @@
+from collections import OrderedDict
+from functools import partialmethod
+
+
+class Restaurant:
+    menu = {'first_course': 'First Course',
+            'second_course': 'Second Course',
+            'dessert': 'Dessert',
+            'beverage': 'Beverage'}
+
+    def __init__(self):
+        self.order = OrderedDict()
+
+    def _add_item(self, key, count, price):
+        if key not in self.menu:
+            raise ValueError(f'{key} is not available in the menu')
+
+        name = self.menu[key]
```

Comment:
transformation to menu name can happen inside calculate_total_cost, since it is actually a view (representation of data)